src/client.cpp

```diff
@@ -500,37 +501,69 @@ void CClient::OnTimerRemoteChanGain()
             Channel.SetRemoteChanGain ( clientChan->iServerChannelID, fGain ); // translate client channel to server channel ID
             bSent = true;
         }
+
+        if ( clientChan->newPan != clientChan->oldPan )
+        {
+            // send new pan and record as old pan
+            float fPan = clientChan->oldPan = clientChan->newPan;
+            Channel.SetRemoteChanPan ( clientChan->iServerChannelID, fPan ); // translate client channel to server channel ID
+            bSent = true;
+        }
     }
 
-    // if a new gain has been sent, reset the range of channel IDs to empty and start timer
+    // if a new gain or pan has been sent, reset the range of channel IDs to empty and start timer
```

Comment:
Doesn't using a single timer mean that sending a single gain and then a single pan could get twice the delay?  Does it matter?  I guess it saves complexity (at the expense of a little readability).

Reply:
I guess an X-Y controller mapped to gain/pain could be very noisy, so this does probably work better.

Reply:
No, it shouldn't. When the single gain has been sent, the timer is started. If a single pan is requested while the timer is running, the new value will get stored locally and then sent as soon as the timer expires. The timer is by default 50ms if ping is less than 25ms, so pretty quick, otherwise it is increased to twice the current ping time.

I didn't feel there was any advantage in having separate gain and pan timers, as almost all users will not be twiddling both at the same time.